nexus/src/app/mod.rs

```diff
@@ -297,6 +302,14 @@ impl Nexus {
             saga_create_tx,
         ));
 
+        // Create a channel for replicating repository artifacts. 16 is a
+        // dubious bound for the channel but it seems unlikely that an operator
+        // would want to upload more than one at a time, and at most have two
+        // or three on the system during an upgrade (we've sized the artifact
+        // datasets to fit at most 10 repositories for this reason).
```

Comment:
Can you give more context to the claim "We've sized the artifact datasets to fit at most 10 repositories"?

Are you saying the M.2 'update' dataset is 10x the size of "a repo"? but which repo?

Reply:
https://github.com/oxidecomputer/omicron/blob/dcc0df384de6b1c708f715f66cc013c6fc3cde5c/sled-storage/src/dataset.rs#L50-L53

All of these numbers are dubious.